community/kernel/src/docs/ops/configuration.asciidoc

```diff
@@ -16,6 +14,8 @@ For additional configuration settings, see:
 * <<config-org.neo4j.ext.udc.UdcSettings>>
 * <<config-org.neo4j.metrics.MetricsSettings>>
 
-
 include::configuration-attributes.asciidoc[]
 
+include::{importdir}/neo4j-server-docs-jar/ops/configuration-attributes.asciidoc[]
+
+include::{importdir}/neo4j-dbms-docs-jar/ops/configuration-attributes.asciidoc[]
```

Comment:
This is probably right, but need to verify the header 'leveloffset' in the output, that the newly included sections are nested at the correct level.